include/core/federated/net_common.h

```diff
@@ -196,7 +196,9 @@ THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #ifndef NET_COMMON_H
 #define NET_COMMON_H
 
+#ifndef PLATFORM_ARDUINO
```

Comment:
This doesn't seem the be the right macro.

Reply:
Arduino doesn't have pthread.h support.

Reply:
That I understand, but this could become a very long list eventually. Don't we have a `THREADED` macro or something along those lines? Maybe @petervdonovan as suggestions?

Reply:
LF_THREADED is set if the threaded runtime is being used.

Reply:
The issue is that Arduino boards have threading but don't use the pthread library

Reply:
Looks like this particular include isn't needed, actually.